pyproject.toml

```diff
@@ -50,6 +50,7 @@ requests = { version = ">=2.21.0", optional = true }
 requests-oauthlib = { version = ">=1.3.0", optional = true }
 six = { version = ">=1.13.0", optional = true }
 httpx = { version = ">=0.23.3", optional = true }
+dnacentersdk = "^2.5.6"
```

Comment:
I think this should be `optional = true`, also, is the actual minimum version 2.5.6 or can we got a bit lower?

Reply:
Corrected the dependency to be optional.